### ISSUE TITLE
instead of putting ffmpeg in the same folder put it in temp folder

### ISSUE THREAD

**Issue by RisedSky** [louislam/lazy-compress-h265#2]
It will prevent the users to copy paste a lot of ffmpeg, the program download ffmpeg in the temp folder using %temp% env, and then use it to convert correctly

**Comment by dave9123** [comment]
Well to make it to the temp folder, he needs even more work.

**Comment by RisedSky** [comment]
> Well to make it to the temp folder, he needs even more work.

Not at all, it justs need some changes in the code to permit it, will you continue maintaining it ? I can do a PR if you want

**Comment by dave9123** [comment]
Do you have a example code for it?

**Comment by RisedSky** [comment]
> Do you have a example code for it?

Yes, but currently my disk isn't available, will take a look when i will have my power supply back (he just died a week ago <,<)

Set a var as the directory of ffmpeg in the temp folder %temp%
Download the ffmpeg into it
Use the %temp%/ffmpeg.exe file to extract and do the process

That's it, not difficult at all, check the file exist on startup, then download and good to go
